apps/web/src/pages/events/list/event-list.page.tsx

```diff
@@ -153,3 +164,19 @@ function EventItem(props: { event: EventsListItem; class?: string }) {
     </li>
   );
 }
+
+function upcomingEventPredicate(event: EventsListItem) {
+  if (!event.date) {
+    return true;
+  }
+
+  if (isToday(event.date)) {
+    return true;
+  }
+
+  return isAfter(event.date, new Date());
+}
+
+function outdatedEventPredicate(event: EventsListItem) {
```

Comment:
`past` seems like a better naming than `outdated`.

Reply:
I agree and changed it.